contracts/TitleEscrowCloneable.sol

```diff
@@ -1,58 +1,18 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "./Initializable.sol";
-import "./Clones.sol";
-import "./ITitleEscrow.sol";
-import "./ITitleEscrowCreator.sol";
-import "./ERC721.sol";
-
-abstract contract HasNamedBeneficiaryInitializable is Context, IHasBeneficiary, Initializable {
-  address public override beneficiary;
-
-    function __initialize__beneficiary(address _beneficiary) internal {
-        beneficiary = _beneficiary;
-    }
-
-  modifier onlyBeneficiary() {
-    require(isBeneficiary(), "HasNamedBeneficiary: only the beneficiary may invoke this function");
-    _;
-  }
-
-  function isBeneficiary() internal view returns (bool) {
-    return _msgSender() == beneficiary;
-  }
-}
-
-abstract contract HasHolderInitializable is Context, IHasHolder, Initializable {
-  address public override holder;
-
-  function __initialize__holder(address _holder) internal {
-    holder = _holder;
-    emit HolderChanged(address(0), _holder);
-  }
-
-  modifier onlyHolder() {
-    require(isHolder(), "HasHolder: only the holder may invoke this function");
-    _;
-  }
-
-  function isHolder() internal view returns (bool) {
-    return _msgSender() == holder;
-  }
-
-  function _changeHolder(address newHolder) internal {
-    require(newHolder != address(0), "HasHolder: new holder is the zero address");
-    emit HolderChanged(holder, newHolder);
-    holder = newHolder;
-  }
-}
+import "./access/HasNamedBeneficiaryInitializable.sol";
```

Comment:
any consequences on the contract size (and deployment cost) ?

Reply:
Did you mean the whether separating the files will increase the contract bytecode size? I think it shouldn't. Just to be safe, I did a quick check on their sizes between before and after this change and they are the same. Here are the numbers for fun:
```
Size in KB.
·························|··············
 |  TitleEscrowCloneable  ·      5.521  │
 ·························|··············
 |  TitleEscrowCloner     ·      0.669  │
 ·························|··············
 |  TradeTrustERC721      ·     10.031  │
 ·------------------------|-------------·
```

Reply:
> Also we will need to republish the contract in etherscan =)

Ohh!😱 We have contracts deployed on mainnet?

Reply:
that's what happens when you split files and interfaces haha :D

Do you think automation could be part of the build on master? post release, only if there is a contract change?

Reply:
Hum not sure you understood me correctly
I think we can leave the library release cycle as is

I was referring to the smart contract releases. I think the ideal solution would be to be able to deploy on Etherscan the contract source when there is any change on it.